src/main/java/sg/ncl/testbed_interface/HttpUtils.java

```diff
@@ -0,0 +1,51 @@
+package sg.ncl.testbed_interface;
+
+/**
+ * Created by dcsjnh on 11/29/2016.
+ *
+ * References:
+ * [1] https://github.com/23/resumable.js/blob/master/samples/java/src/main/java/resumable/js/upload/HttpUtils.java
+ */
+public class HttpUtils {
+
+    public static boolean isEmpty(String value) {
+        return value == null || "".equals(value);
+    }
+    /**
+     * Convert String to long
+     * @param value
+     * @param def default value
+     * @return
+     */
+    public static long toLong(String value, long def) {
+        if (isEmpty(value)) {
+            return def;
+        }
+
+        try {
+            return Long.valueOf(value);
+        } catch (NumberFormatException e) {
+            e.printStackTrace();
+            return def;
+        }
+    }
+
+    /**
+     * Convert String to int
+     * @param value
+     * @param def default value
+     * @return
+     */
+    public static int toInt(String value, int def) {
+        if (isEmpty(value)) {
+            return def;
+        }
+        try {
+            return Integer.valueOf(value);
+        } catch (NumberFormatException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)